Spreadsheets (#1783)
enkidevs/curriculum: excel/beginner/charts/why-are-charts-useful.md

---
author: Stefan-Stojanovic

levels:
  - beginner

aspects:
  - new
  - workout

type: normal

category: how to

---

# Why are charts useful?

---
## Content

> A good example of why charts are useful is because they can be used to detect patterns and understand results in a more clear way.

Reviewer: Maybe we could also add the reference to the famous quote "A picture tells a 1000 words"

They are good for understanding large quantities of data with ease and speed.

> A single chart can easily represent thousands of data.

Reviewer: Combine this sentence with the one above it? Something like:

> They are good for understanding large quantities of data that would otherwise be difficult to individually analyze. A single chart can represent thousands of data points.

Each type of chart also "tells a story".

> This is because upon creating a chart with large data, you can instantly see patterns and visualize the data and use it to determine any negative or positive information regarding the data.

Reviewer: I'd simplify this sentence a little bit.

Maybe:

```suggestion
This is because visualizations can quickly demonstrate patterns and allow us to draw conclusions and determine any negative or positive outcomes about our data.
```

---
## Practice

Which of these is not true?

???

* Charts were used before computers were made and now have no meaning.

> * A single chart can easily represent thousands of data.

Reviewer: ```suggestion
* A single chart can represent thousands of data points.
```

* Charts can be used to detect patterns and understand results in a more clear way.

> * They good for fast analysis of data.

Reviewer: They gucci 😆 

```suggestion
* Charts are good for fast analysis of large amounts of data.
```
